themes/custom/droopler_theme/src/components/d-p-side-embed/_d-p-side-embed.scss

```diff
@@ -11,6 +11,7 @@
 
   --media-width: 100%;
   --media-height: 100%;
+  --cta-links-alignment: left;
```

Comment:
Do we have to define it here if `left` is the default?

In most cases, alignment is `center`. Maybe, should be better to have it as default and remove it from the places where the default should be applied? What do you think?

Reply:
You're right. Changed the default to `center` and removed unnecessary overrides.